### PR TITLE
Mobile Portfolio Project

### PR DESCRIPTION
**Pull request by lilyfrancis**
This is a mobile version portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background. In this Project I added the following:

- Setup Linters
- Index File
- A CSS style file
- A ReadMe File

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,28 +12,191 @@
...
+          Glad to see you!</h1>
+      </div>
+      <div class="description">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+          and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</p>
+      </div>
+      <div class="connect">
+        <a href="#" class="connect">LET’S CONNECT</a>
+      </div>
```
Comment (Shaher-11) [inline index.html]: Kindly fix the styling of the part to be similar to the original design shown below
#### Original design
right1

#### Your Project
wrong1

File: index.html [hunk 2]
```diff
@@ -12,28 +12,191 @@
...
+          <li class="social-icon">
+            <a href="https://github.com/ifnotlily/">
+              <img src="images/github.svg" alt="Lilian's Github" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+  </section>
+  <section class="section-grid">
```
Comment (Shaher-11) [inline index.html:80]: You need to fix the styling for the project cards as the difference shown below
#### Original design
right2

#### Your Project
wrong2

File: index.html [hunk 3]
```diff
@@ -62,6 +225,27 @@ <h1>I’m Lilian
...
+        </li>
+
+
+        <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+
+
+        <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
+      </ul>
+
+
```
Comment (Shaher-11) [inline index.html]: Reduce the space between the got resume btn and the languages also add background color to each on of the languages  and align the text to the icon
#### Original design
right3

#### Your project
wrong3

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @ifnotlily,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] You must stick to the design as much as possible using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) and make sure the new changes do not affect the work that's been already done
- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: Resolved

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:80]: Resolved

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: Resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Lilian, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it. 👍.
Congratulations! 🥂. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Kindly consider:
- [x] OPTIONAL
Please bear in mind that you, for best practices and following the project requirements, should have followed the [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in the aspects (colors, backgrounds, font, font size, font weight, margin, and padding).


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Pull request merged**